core-web/libs/edit-content/src/lib/components/dot-edit-content-form/dot-edit-content-form.component.scss

```diff
@@ -5,6 +5,12 @@
     grid-template-columns: 1fr 16rem;
     gap: $spacing-4;
     padding: $spacing-4;
+    padding-bottom: 0;
+    height: 100%;
+
+    form {
+        overflow: auto;
+    }
```

Comment:
Why?

Reply:
the form is not scrolling

Reply:
the form was not scrolling without this

Reply:
Hacky... we need to do something different because the sidebar will need scroll as well.